src/init.cpp

```diff
@@ -1696,6 +1692,7 @@ bool AppInitMain(const CoreContext& context, NodeContext& node, interfaces::Bloc
     node.chainman = &g_chainman;
     ChainstateManager& chainman = *Assert(node.chainman);
 
+    assert(!::governance);
```

Comment:
`assert(!::governance);` means the opposite, it's equivalent to `::governance == nullptr`. It is a check to make sure we're not overwriting an existing pointer (which would indicate double initialization). This convention exists throughout `init.cpp`. Explicit comparison would break from that convention.